mkdocs.yml

```diff
@@ -14,8 +14,22 @@ theme:
   favicon: images/icon-square.png
   logo: images/icon-square.png
   palette:
```

Comment:
The light/dark mode toggle doesn't seem to work on the recipe pages...

* https://asemy.github.io/okio/recipes/
* https://asemy.github.io/okio/java_io_recipes/

Might be a bug? Not sure.

Reply:
In the console logs I see an error in `index.ts:77`

```
index.ts:77 Uncaught ReferenceError: Missing element: expected ".tabbed-labels" to be present
    at K (index.ts:77:11)
    at mi (index.ts:135:21)
    at index.ts:114:21
    at Array.map (<anonymous>)
    at di (index.ts:114:8)
    at bundle.ts:216:16
    at Array.map (<anonymous>)
    at bundle.ts:216:6
    at e._subscribe (defer.ts:55:15)
    at e._trySubscribe (Observable.ts:245:19)
```

Reply:
Might be this issue https://github.com/squidfunk/mkdocs-material/issues/3330#issuecomment-991541837

https://squidfunk.github.io/mkdocs-material/upgrade/#pymdownxtabbed

Reply:
This should be fixed now! Although I can't test the PR branch directly. The live demo should match the feature branch: https://asemy.github.io/okio/